### PR TITLE
Add a version in package.json

### PR DESCRIPTION
**Pull request by seanballais**
In response to issue #3712.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "electron",
+  "version": "v0.35.0",
```
Comment (xywz) [inline package.json]: Current version is 0.35.4. Remove "v" to be consistent with `process.versions['electron']` and electron-prebuilt.

**Comment by xywz** [comment]
Something should probably be added to [bump-version.py](https://github.com/atom/electron/blob/master/script/bump-version.py).

**Comment by seanballais** [comment]
@xywz What does *bump-version.py* do? Does it handle package.json too?

**Comment by zcbenz** [comment]
> @xywz What does bump-version.py do? Does it handle package.json too?

It changes the version strings in source code to the latest one and it doesn't handle the `package.json`.

**Comment by seanballais** [comment]
>>@xywz What does bump-version.py do? Does it handle package.json too?

>It changes the version strings in source code to the latest one and it doesn't handle the package.json.

@zcbenz Is `package.json` autogenerated?

File: package.json (hunk 1 above)
Comment (seanballais) [inline package.json]: I think I should probably add something to `bump-version.py` instead. Thanks for the info by the way.

**Comment by seanballais** [comment]
I modified `bump-version.py` to add `version` in `package.json`. I will be committing it soon.

File: package.json (hunk 1 above)
Comment (xywz) [inline package.json]: You should do both, since `bump-version.py` modifies existing strings.

Comment (seanballais) [inline package.json]: Ah. Did the changes. Pushing the commits now.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "electron",
+  "version": "0.35.4",
```
Comment (xywz) [inline package.json]: 0.36.0 now.

**Comment by seanballais** [comment]
CI build failed. Did my code break something?

**Comment by seanballais** [comment]
What do you guys think?

**Comment by zcbenz** [comment]
Looks good to me, thanks! It has been committed at 2d2ad0d33bdfc7d89fc2e68deb363fd846492328.

**Comment by seanballais** [comment]
Ah. Thanks! :+1:

**Pull request closed without merging**